internal/conv/conv.go

```diff
@@ -39,6 +41,8 @@ func String(a any) (string, error) {
 		return strconv.FormatFloat(xt, 'f', -1, 64), nil
 	case bool:
 		return strconv.FormatBool(xt), nil
+	case []byte:
+		return string(xt), nil
```

Comment:
I think this could use some docs.  the returned string isn't guaranteed to be valid UTF8 (won't be legible printed and will be rejected if attempting to use where a `TEXT` type is expected by postgresql).
What case does this support?  It's not obvious since this is in the `internal/conv` package so there could be many consumers with different needs.

Depending on the intent, maybe we want to employ https://pkg.go.dev/unicode/utf8#Valid before casting with no conversion.

Reply:
Ah that's a good catch. The intent is to convert to a postgres TEXT type